smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/paginated/paginated-deeply-nested.errors

```diff
@@ -1,2 +1,2 @@
-[WARNING] ns.foo#DeeplyNested: paginated trait `items` contains a path with more than two parts, which can make your API cumbersome to use | PaginatedTrait
-[WARNING] ns.foo#DeeplyNested: paginated trait `outputToken` contains a path with more than two parts, which can make your API cumbersome to use | PaginatedTrait
+[WARNING] ns.foo#DeeplyNested: paginated trait `items` contains a path with more than two parts, which can make your API cumbersome to use | PaginatedTrait.items.DEEPLY_NESTED
```

Comment:
Why DEEPLY_NESTED and not DeeplyNested? I'd also think it might be better to do this as `PaginatedTrait.DeeplyNested.items` so you could ignore all deeply nested violations for a single event.

Reply:
In general, I wanted to use constant/enum casing for these sort of things because in some of the validators, it appears ambiguous at first glance if it is supposed to be a shape Id or a signal-type thing as in the case of DEEPLY_NESTED. I also thought that it was better to scope any suppressions to each trait, but I'll change the ordering on the next commit

Reply:
I think a good pattern for us to follow would be Validator.[Category].[more-specific-categories].[specific shape ID or member], where everything after Validator is optional.

As for casing, I think exposing ENUM_CASING for some things but not others, including the main validator name, adds inconsistency to the event IDs and exposes the implementation detail that Java uses all caps enum member names. Is it difficult to get the casing right or something? Maybe you could just use the desired casing in the enum names rather than ALL_CAPS member names in the Java enums (if they're private enums it doesn't really matter).

Reply:
Since we already have validation event IDs, and they don't use SCREAM_CASE casing, I'd rather we avoided mixing it into event IDs.

Reply:
ok, will change to camel case 👍